ProjectSidewalk/SidewalkWebpage
# Resolves #965: Generic context menu close event

Resolves #965 

This is a followup to https://github.com/ProjectSidewalk/SidewalkWebpage/pull/1465, adding an additional generic ContextMenu_Close event when contextMenu.hide is called. This was necessary because some places in the code were calling contextMenu.hide without sending a close event.

To test:
- Ensure that temporary label IDs are always null when the context menu is closed
- Ensure that temporary label IDs are always non-null when the context menu is open
